src/arch/arm/tlb.hh

```diff
@@ -167,7 +185,7 @@ class TLB : public BaseTLB
     /** Lookup an entry in the TLB
      * @return pointer to TLB entry if it exists
      */
-    TlbEntry *lookup(const Lookup &lookup_data);
+    TlbEntry *lookup(Lookup lookup_data);
```

Comment:
Just because we want to make a copy of the data anyway since we iterate over different page sizes withing the lookup method and we need a non cont parameter

Reply:
Oh, ok. I thought it would be something related to the fact that const references may generate bugged code (https://stackoverflow.com/a/4705871)